metadata.json

```diff
@@ -41,7 +58,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.10.0 < 8.0.0"
+      "version_requirement": ">= 6.2.0 < 8.0.0"
```

Comment:
Is there a specific reason to go to 6.2.0? We usually go to 6.1.0 because that has the systemctl refresh resource embedded.

Reply:
Not at all, thought 6.2.0 was the one to go for for some reason.